resources/fetching/fetchers/gcp/assets_fetcher.go

```diff
@@ -137,20 +136,33 @@ func (r *GcpAsset) GetMetadata() (fetching.ResourceMetadata, error) {
 	}
 
 	return fetching.ResourceMetadata{
-		ID:      r.Asset.Name,
-		Type:    r.Type,
-		SubType: r.SubType,
-		Name:    getAssetResourceName(r.Asset),
-		Region:  region,
+		ID:        r.Asset.Name,
+		Type:      r.Type,
+		SubType:   r.SubType,
+		Name:      getAssetResourceName(r.Asset),
+		Region:    region,
```

Comment:
we should remove `region` from `fetching.ResourceMetadata` and send it as part of `ecs.cloud`